pytissueoptics/scene/logger/logger.py

```diff
@@ -16,47 +17,66 @@ class Segment:
     end: Vector
 
 
-class Logger:
+@dataclass(frozen=True)
+class InteractionKey:
+    solidLabel: str
+    surfaceLabel: str = None
+
+
+class InteractionData:
```

Comment:
Isn't this a @dataclass too?

Reply:
Possible https://stackoverflow.com/questions/52063759/passing-default-list-argument-to-dataclasses, but I get why you didn't. moving on/